### DISCUSSION TITLE
Generated PDF font weight isn’t working

### DISCUSSION THREAD

**Discussion by Abigailvantol** [spatie/laravel-pdf#166] [Q&A]
Hi everyone, 

does anyone know why font weight isn’t working? And how I can get it to work?

This is when the PDF is generated :
![PDF styling](https://github.com/user-attachments/assets/8030a4ed-a805-4929-8e43-6cd72666d7a1)

And this is the result from the Blade itself:
![Blade styling](https://github.com/user-attachments/assets/35771c00-fb49-470e-a475-343b07aa2970)

Code:
```
<p>The quick brown fox jumps over the lazy dog.</p>
<p class="font-light">The quick brown fox jumps over the lazy dog.</p>
<p class="font-light text-sm"><span class="font-semibold">The quick brown</span> fox jumps over the lazy dog.</p>
<p class="font-light"><span class="font-semibold">The quick brown</span> fox jumps over the lazy dog.</p>
```

**Comment by ArielMejiaDev** [comment]
@Abigailvantol did you set this in blade header? of footer? css only works for the PDF content blade view, probably this helps or not, sorry that I cannot go into more detailed explanation, but basically laravel/pdf use browsershot and browsershot use puppeteer and it does not support css for headers and footers

**Reply by Abigailvantol** [reply]
No, I don't use a header or footer.

```
return pdf()
    ->withBrowsershot(function (Browsershot $browsershot) {
        $browsershot->noSandbox();
    })
    ->view('pdf',  $document->data)
    ->name($filename);
```
